docs/topics/spiders.rst

```diff
@@ -692,7 +692,7 @@ SitemapSpider
 
     .. method:: sitemap_filter(entries)
 
-        This is a filter funtion that could be overridden to select sitemap entries
+        This is a filter function that could be overridden to select sitemap entries
```

Comment:
May I suggest an additional change?

```suggestion
        This is a filtering function that could be overridden to select sitemap entries
```

Reply:
Both ways of writing it look fine to me.